client/my-sites/email/email-forwarding/index.jsx

```diff
@@ -34,7 +36,10 @@ class EmailForwarding extends Component {
 	};
 
 	render() {
-		const { selectedDomainName, translate } = this.props;
+		const { compact = false, selectedDomainName, translate } = this.props;
+		if ( compact ) {
+			return this.renderContent();
+		}
```

Comment:
This code path needs to make sure that we include `<QueryEmailForwards domainName={ selectedDomainName } />` to get `emailForwards` populated.

Reply:
But when we show this form, I think we are pretty sure that there aren't any other forwards for this domain, isn't it?

Reply:
Right now, we're only showing this form from within `EmailProvidersComparison`, but we plan to support using the form for adding new email forwards from a different context.
We should either have the component query for the email forwards, or ensure that the data is specified in the API for the component.